app/views/idv/in_person/state_id.html.erb

```diff
@@ -111,11 +111,12 @@
           ) %>
     </div>
   <% end %>
-  <div class="tablet:grid-col-8 margin-bottom-5">
+  <div class="margin-bottom-5">
     <%= render ValidatedFieldComponent.new(
           name: :state_id_number,
           form: f,
           hint: t('in_person_proofing.form.state_id.state_id_number_hint'),
+          hint_html: { class: 'hint-grid-col-10' },
```

Comment:
Do we need a new class or can we use existing one?

```suggestion
          hint_html: { class: 'tablet:grid-col-10' },
```

(Also noting because "1024px" is not a standard breakpoint width used elsewhere in the application)

Reply:
@aduth This class should be applied to desktop only. What value would you use? I am seeing 1024px in the code a lot. I am not seeing a similar set up for desktop so went with a new class. I can set up desktop\:grid-col-10 though if that is preferred.

Reply:
"tablet" is the standard breakpoint size, which is 640px. Is there a reason is must only apply at desktop and not use the standard breakpoint instead?

Reply:
It was in our discussion as a breakpoint. We are just trying to breaking the line so that the characters are not on the second line alone with no text. It is achieving the same thing and will keep with the standard breakpoint size. Thanks